index.js

```diff
@@ -0,0 +1,197 @@
+/** @format */
+
+function addItem() {
+  const userInputValue = document.getElementById('input').value;
+
+  const item = {
+    description: userInputValue,
+    completed: false,
+    index: 0,
+  };
+
+  const itemArray = [];
+
+  if (localStorage.getItem('items').length) {
+    const currentItems = JSON.parse(localStorage.getItem('items'));
+
+    if (currentItems.length > 0) {
+      currentItems.forEach((item) => {
+        itemArray.push(item);
+      });
+    }
+  }
+
+  if (item.description.trim().length > 0) {
+    item.index = itemArray.length;
+    itemArray.push(item);
+  }
+
+  localStorage.setItem('items', JSON.stringify(itemArray));
+
+  reOrder();
+  loadItems();
+
+  document.getElementById('input').value = '';
+}
+
+function updateItem() {
+  const items = JSON.parse(localStorage.getItem('items'));
+
+  const updatedValue = document.getElementById('updateId').value;
+  const itemIndex = document
+    .getElementById('updateId')
+    .getAttribute('elementid');
+
+  items[itemIndex].description = updatedValue;
+
+  localStorage.setItem('items', JSON.stringify(items));
+
+  loadItems();
+
+  document.getElementById('formCreate').style.display = 'block';
+  document.getElementById('formUpdate').style.display = 'none';
+}
+
+function checkStatus() {
+  const items = JSON.parse(localStorage.getItem('items'));
+  const checkBoxes = document.querySelectorAll('.check');
+
+  checkBoxes.forEach((box, index) => {
+    const paragraph = box.nextElementSibling;
+
+    box.checked = items[index].completed;
+    if (items[index].completed) {
+      paragraph.style.textDecoration = 'line-through';
+      paragraph.style.color = '#999';
```

Comment:
- [ ] Try to remove your styles from your HTML file instead you can use CSS classes to style your web content. you can use dom-manipulation like `HTMLElement.classList.add('className)`